wandb/sdk/launch/utils.py

```diff
@@ -595,8 +595,9 @@ def get_kube_context_and_api_client(
             raise LaunchError(f"Specified context {context_name} was not found.")
         else:
             context = active_context
-
-        kubernetes.config.load_kube_config(config_file, context["name"])
+        kubernetes.config.load_kube_config(
+            config_file, context["name"]
```

Comment:
For this we can use the `get_module` util to throw a good exception to install `wandb[launch]` if it fails